internal/controller/server_controller.go

```diff
@@ -638,6 +641,10 @@ func (r *ServerReconciler) ensureServerPowerState(ctx context.Context, log logr.
 		server.Spec.Power == metalv1alpha1.PowerOff {
 		powerOp = powerOpOff
 	}
+	if server.Status.State == metalv1alpha1.ServerStateDiscovery &&
```

Comment:
Can we think of a way to keep local dev specific logic out of the main reconciliation loop?

Reply:
It's tricky because mockup server is always powered on real fast, and even `--enforce-first-boot` does not help. I have pushed a rework that is specific to this dev UUID. Works for now but we may want to revisit if we even want to spin up multiple mockups servers in kube.

Reply:
actually found a simple way - just use `SetPXEBootOnce` instead of `PowerOn`. That does the trick

Reply:
Ok, it seems as this will take a while until we can use `go install` for `kubectl` https://github.com/kubernetes/kubectl/issues/1140